src/components/CheckingAccountCreation.jsx

```diff
@@ -0,0 +1,118 @@
+import { useState } from "react";
+import { postCheckingAccounts } from "../api";
+
+const initialState = {
+    accountType:'SCK',
+    accountOwnership:'IND',
+    accountName:'',
+    accountOpening:''
+}
+
+export const CheckingAccountCreation = () => {
+
+    const [account,setAccount] = useState(initialState); 
+    const [successAlert,setSuccessAlert] = useState(false); 
+    const [depositError,setDepositError] = useState(false); 
+
+    const handleInputChange = ({target}) =>{
+
+        setAccount({
+            ...account,
+            [target.name]:target.value
+        })
+
+    }
+    
+
+    const resetForm = ()=>{
+       setAccount(initialState); 
+    }
+    
+
+    const handleFormSubmit = async(e) =>{
+        e.preventDefault(); 
+        
+        if(Number(account.accountOpening || account.accountOpening === '') < 25){
+            return setDepositError(true); 
+        }
+
+
+        const response = await postCheckingAccounts(account)
+
+        if(response.status === 200){
+            setSuccessAlert(true);
+            resetForm(); 
+            
+        }
+    }
+
+    return (
+        <div style={{ paddingTop: 120 }}>
+
+            <form onSubmit={handleFormSubmit} className="create-account-form">
+                <h3>New Checking Account</h3>
+                <div>
+                    <strong className="d-block ms-3 mb-2">Select Checking Account Type</strong>
+                    <div className="d-inline">
+                        <input onChange={handleInputChange} name="accountType" className="form-check-input ms-3" type="radio" value="SCK" id="flexCheckDisabled" checked={account.accountType === "SCK"} />
+                        <label className="ms-1"> Standard checking</label>
+
+                    </div>
+
+                    <div className="d-inline">
+                        <input onChange={handleInputChange} name="accountType" className="form-check-input ms-3" type="radio" value="ICK" id="flexCheckDisabled" checked={account.accountType === "ICK"} />
+                        <label className="ms-1"> Interest checking</label>
+
+                    </div>
```

Comment:
You can create a `Radio` component for all these inputs of type radio and reduce code lines.

Reply:
Thats a great idea, I'll do that and I'll pass the values as props